infra/aws/terraform/prow-build-cluster/variables.tf

```diff
@@ -14,12 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-variable "eks_admins" {
+variable "eks_cluster_admins" {
   type        = list(string)
-  description = "List of maintainers that have administrator access to the account and cluster"
+  description = "List of maintainers that have administrator access to the cluster."
   default     = []
 }
 
+variable "eks_cluster_viewers" {
+  type        = list(string)
+  description = "List of maintainers that have view access to the account and cluster."
```

Comment:
Is this description up to date? I think that view access to the account depends on if you have EKSInfraViewer role attached, and that's handled in the IAM module.